.ci/run.sh

```diff
@@ -32,7 +32,7 @@ function cleanup {
 
 trap cleanup EXIT
 
-if [ "$ID" == "ubuntu" ] && [ "$VERSION_ID" == "17.04" ]; then
+if [ "$ID" == "ubuntu" ] && ( [ "$VERSION_ID" == "17.04" ] || [ "$VERSION_ID" == "17.10" ] ); then
```

Comment:
I think it would be more "future-proof" if you make this more general. Something like:

```
MAJOR=$(echo "$VERSION_ID"|cut -d\. -f1)

# Only use devicemapper for Ubuntu 17.* and newer.
if [ "$ID" == "ubuntu" ] && [ "$MAJOR" -ge 17 ]; then
    :
    :
```